.github/workflows/ci.yml

```diff
@@ -170,6 +170,33 @@ jobs:
               '\.py$|.github/workflows/|^Dockerfile|^scripts'
           echo "::set-output name=count::$?"
         id: pyfiles
+
```

Comment:
I think we are going to split the ci.yml into separate workflows soon. So maybe a good idea will be to split out open-apis already to a separate openapi.yml file ?